### PR TITLE
Feature/eui 1492 reduce docker

### PR DESCRIPTION
**Pull request by udaydenduluri33**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: config/default.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
```
Comment (paddybasi) [inline config/default.json]: default.json should have 'default' values (that can be hardcoded strings)

File: charts/xui-mo-webapp/values.aat.template.yaml [hunk 2]
```diff
@@ -4,6 +4,17 @@ nodejs:
   environment:
     IDAM_URL: https://idam-api.aat.platform.hmcts.net
     S2S_URL: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
+    NODE_TLS_REJECT_UNAUTHORIZED: 1
     S2S_SECRET: ${S2S_SECRET}
     IDAM_SECRET: ${IDAM_SECRET}
-    NODE_TLS_REJECT_UNAUTHORIZED: 1
+    CCD_DATA_API_SERVICE: http://ccd-data-store-api-demo.service.core-compute-demo.internal
```
Comment (paddybasi) [inline charts/xui-mo-webapp/values.aat.template.yaml]: need to move these urls to values.yaml in a generic way

File: charts/xui-mo-webapp/values.preview.template.yaml [hunk 3]
```diff
@@ -4,6 +4,17 @@ nodejs:
   environment:
     IDAM_URL: https://idam-api.aat.platform.hmcts.net
     S2S_URL: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
+    NODE_TLS_REJECT_UNAUTHORIZED: 0
     S2S_SECRET: ${S2S_SECRET}
     IDAM_SECRET: ${IDAM_SECRET}
-    NODE_TLS_REJECT_UNAUTHORIZED: 0
+    CCD_DATA_API_SERVICE: http://ccd-data-store-api-demo.service.core-compute-demo.internal
```
Comment (paddybasi) [inline charts/xui-mo-webapp/values.preview.template.yaml]: same again, move urls to values.yaml

File: config/default.json (hunk 1 above)
Comment (udaydenduluri33) [reply config/default.json]: Done

File: charts/xui-mo-webapp/values.aat.template.yaml (hunk 2 above)
Comment (udaydenduluri33) [reply charts/xui-mo-webapp/values.aat.template.yaml]: For later as agreed

File: charts/xui-mo-webapp/values.preview.template.yaml (hunk 3 above)
Comment (udaydenduluri33) [reply charts/xui-mo-webapp/values.preview.template.yaml]: For later as agreed

File: api/local.ts [hunk 4]
```diff
@@ -18,39 +37,68 @@ import routes from './routes'
...
+      path: getConfigValue(NOW) ? '/tmp/sessions' : '.sessions',
     }),
   })
 )
 
 /**
  * Used Client side
  */
-if (config.proxy) {
+if (getConfigValue(PROXY_HOST)) {
```
Comment (paddybasi) [inline api/local.ts]: you don't need this condition as it's already there in the method

Comment (udaydenduluri33) [reply api/local.ts]: Done

**Pull request merged**